packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -58,15 +58,9 @@ export const AddMetadataButton = (
     setOpen((prevOpen: boolean) => !prevOpen);
   };
 
-  const sortedSchema = props.schemas?.sort((a, b) => {
-    if (a.name > b.name) {
-      return 1;
-    } else if (a.name < b.name) {
-      return -1;
-    } else {
-      return 0;
-    }
-  });
+  const sortedSchema = props.schemas?.sort((a, b) =>
+    a.title.localeCompare(b.title)
```

Comment:
Using [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#sort_an_array) to enable case-insensitive sorting.